openmc/deplete/abc.py

```diff
@@ -174,17 +174,23 @@ class ReactionRateHelper(ABC):
     Reaction rates are passed back to the operator for be used in
     an :class:`openmc.deplete.OperatorResult` instance
 
+    Parameters
+    ----------
+    n_nucs : int
+    n_react : int
+        Number of burnable nuclides and reactions tracked
+        by :class:`openmc.deplete.Operator`.
```

Comment:
I don't think this will render correctly in the generated documentation. Put a description for `n_nucs` separately.

Reply:
Yeah, it doesn't really look great. Easy fix though!